src/test/java/org/alliancegenome/curation_api/bulkupload/fms/PhenotypeAnnotationBulkUploadFmsITCase.java

```diff
@@ -194,7 +236,16 @@ public void agmPhenotypeAnnotationAddSecondaryAnnotationFields() throws Exceptio
 	@Test
 	@Order(5)
 	public void allelePhenotypeAnnotationAddSecondaryAnnotationFields() throws Exception {
-		//TODO: add test for addition of secondary gene annotation
+		checkSuccessfulBulkLoad(phenotypeAnnotationBulkPostEndpoint, phenotypeAnnotationTestFilePath + "AS_01_add_secondary_allele_annotation_to_primary_allele_annotation.json", 3);
+		
+		RestAssured.given().
+			when().
+			get(phenotypeAnnotationGetEndpoint + allelePaId).
+			then().
+			statusCode(200).
+			body("entity.phenotypeAnnotationSubject.modEntityId", is(allele)).
+			body("entity.inferredGene.modEntityId", is(gene)).
+			body("entity.assertedAllele.modEntityId", is(allele));
```

Comment:
I'm confused as to how this test is passing.  It shouldn't be possible for an AllelePhenotypeAnnotation to have an assertedAllele, and the corresponding test file should be throwing an invalid exception.  Also, I don't see a gene referenced in the test file for the inferredGene field.

Reply:
Sorry for the confusion. Will make necessary changes.